site/README.md

```diff
@@ -49,10 +49,14 @@ To see changes in Javadoc, run:
 
 ### Publishing
 
-After site changes are committed, you can publish the site with this command:
+After site changes are committed, you can publish them by running the following sequence of commands from the `$projectRoot/site` directory:
 
 ```
-mkdocs gh-deploy
+rm -r site
+mkdir site
+cp ../.asf.yaml site/
+mkdocs gh-deploy --dirty
 ```
+This assumes that the Apache remote is named `apache` and will push to the `asf-site` branch. To use a different remote add `-r <remote-name>` to the `mkdocs` command.
```

Comment:
Please add a blank line to separate the preformatted content from the next paragraph, like before.